docker-compose.yml

```diff
@@ -4,9 +4,9 @@ services:
     container_name: buildingmotif-db
     image: postgres
     environment:
-       - POSTGRES_DB=${POSTGRES_DB}
-       - POSTGRES_USER=${POSTGRES_USER}
-       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+       - POSTGRES_DB
+       - POSTGRES_USER
+       - POSTGRES_PASSWORD
```

Comment:
Small but docker compose will automatically pull variables from the environment.
https://docs.docker.com/compose/environment-variables/set-environment-variables/